pemFioi/quickAlgo/i18n.js

```diff
@@ -158,6 +169,11 @@ var quickAlgoLanguageStrings = {
          read: "Reading",
          print: "Writing",
       },
+      exerciseTypeAbout: {
+         default: "Subject powered by <a href='http://www.france-ioi.org/'>France-IOI</a>",
```

Comment:
Same modification to apply to other translations.

Reply:
Hello, for "Task" what would be the translation in French? The current one in french is "Sujet propulsé par France-IOI", you asked to replace "Subject" in the english version but I don't know what would be the best for the French one.

I also asked @mathias-hiron for the "default" translation (translation for all other exercices than Quick-Pi) what he think of it, because I have only asked him about the about section for Quick-Pi and I don't know if he has something different in mind for the default section

Reply:
Hello, I think "Sujet" is fine in French. Note that "Subject" is not a good translation of "Sujet" (for this meaning of "Sujet"), hence I suggested to change it.

Another word we could use in French is "Tâche", but I really think "Sujet" is good.

Reply:
Thank you, I have changed "Subject" to "Task"